meson.build

```diff
@@ -350,7 +350,10 @@ lib_udev = dependency(
   required : get_option('systemd'))
 conf.set('HAVE_LIBUDEV', lib_udev.found() ? 1 : false)
 
-lib_crypt = cc.find_library('crypt')
+lib_crypt = cc.find_library('crypt', required : get_option('build-newgrp'))
```

Comment:
`lib_crypt = cc.find_library('crypt', required : get_option('build-newgrp') or get_option('build-sulogin'))`?

Reply:
Aha! I first tried this, but had to use `.enabled()` on the options. I decided to keep in the style of following checks, but this is probably better.

Reply:
Now, looking at the output, I remember why I changed this to follow the other paradigm. Using `lib_crypt = cc.find_library('crypt', required : get_option('build-newgrp').enabled() or get_option('build-sulogin').enabled())` yields this output:

```
Library crypt found: YES
```

Where as the current method yields:

```
Library crypt skipped: feature build-newgrp disabled
Library crypt skipped: feature build-sulogin disabled
```

So it appears using a feature option disables detecting the library at all where a `bool` value of `false` will still attempt to find the library. I don't think there is a way to supply multiple feature options to the `required` argument, but I could be missing something.

Reply:
Both methods work, so I'll leave this as is for now to avoid attempting to find the library at all.

Reply:
@t-8ch What do you think?

Reply:
I don't see a big problem with searching for the library even if not needed.
But as the search for pam just below is using the same pattern, let's go with your solution as is.